src/cli/ui.rs

```diff
@@ -111,17 +111,23 @@ pub fn display_program(program: &Program) {
     println!();
 }
 
+pub fn clear_console() {
+    print!("\x1B[2J\x1B[1;1H");
```

Comment:
This works on Linux, but does this also work on Windows/Mac?

Reply:
It works for Windows and Linux can't test for Mac

Reply:
I am pretty sure this also works on mac os. Windows only supports it in "recent" versions, but I guess that is good enough. :sweat_smile: